providers/edge/README.rst

```diff
@@ -0,0 +1,82 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+ .. NOTE! THIS FILE IS AUTOMATICALLY GENERATED AND WILL BE OVERWRITTEN!
+
+ .. IF YOU WANT TO MODIFY TEMPLATE FOR THIS FILE, YOU SHOULD MODIFY THE TEMPLATE
+    `PROVIDER_README_TEMPLATE.rst.jinja2` IN the `dev/breeze/src/airflow_breeze/templates` DIRECTORY
+
+
+Package ``apache-airflow-providers-edge``
+
+Release: ``0.10.1pre0``
+
+
+Handle edge workers on remote sites via HTTP(s) connection and orchestrates work over distributed sites
+
+
+Provider package
+----------------
+
+This is a provider package for ``edge`` provider. All classes for this provider package
+are in ``airflow.providers.edge`` python package.
+
+You can find package information and changelog for the provider
+in the `documentation <https://airflow.apache.org/docs/apache-airflow-providers-edge/0.10.1pre0/>`_.
+
+Installation
+------------
+
+You can install this package on top of an existing Airflow 2 installation (see ``Requirements`` below
+for the minimum Airflow version supported) via
+``pip install apache-airflow-providers-edge``
+
+The package supports the following python versions: 3.9,3.10,3.11,3.12
+
+Requirements
+------------
+
+==================  ==================
+PIP package         Version required
+==================  ==================
+``apache-airflow``  ``>=2.10.0``
+``pydantic``        ``>=2.10.2``
+``retryhttp``       ``>=1.2.0``
+==================  ==================
+
+Cross provider package dependencies
+-----------------------------------
+
+Those are dependencies that might be needed in order to use all the features of the package.
+You need to install the specified provider packages in order to use them.
+
+You can install such cross-provider dependencies when installing from PyPI. For example:
+
+.. code-block:: bash
+
+    pip install apache-airflow-providers-edge[common.compat]
```

Comment:
Why common.compat is optional ? It sounds strange. I would expect compat to be required dependency.

Reply:
I don't know. I just can say: `NOTE! THIS FILE IS AUTOMATICALLY GENERATED AND WILL BE OVERWRITTEN!` :-D

Reply:
Edge does not need the compat. Does the template need to have more logic or to we add the common-compat dependency per default in future?

Reply:
HMMMMMMM... there must be some bug then when generating pyproject.toml. I honestly have no idea where this one came from then :(

Reply:
Ok. I know where it comes from.

This is in example_dags in the edge provider: 

```
try:
    from airflow.operators.python import PythonOperator  # type: ignore
except ImportError:
    from airflow.providers.common.compat.standard.operators import PythonOperator
```

We should remove example dags from "cross-provider-dependencies" calculations.

Reply:
PR is coming

Reply:
https://github.com/apache/airflow/pull/45784

Yep - after the fix, "comon.compat" is not detected as cross-provider dependncy of edge any more. 

BTW. That was the only case - there was some conditional code to remove example_dags for "standard" provider from this cross-provider calculation but this is a general rule that should be applied to all providers.